src/app/organizations/organization/organization.component.html

```diff
@@ -23,46 +23,62 @@
 <ng-template #placeholder>
   <div class="pt-2"></div>
   <div class="container" *ngIf="!(organization$ | async)">Organization not found</div>
-<div class="container" *ngIf="(organization$ | async) as org" fxLayout="column" fxLayoutGap="10px">
-  <mat-card fxFlex class="my-2">
-    <mat-card-header>
-      <mat-card-title>
-        <h1>{{org.name}}</h1>
-      </mat-card-title>
-      <mat-card-subtitle>
-        {{org.topic}}
-      </mat-card-subtitle>
-    </mat-card-header>
-    <mat-card-content>
-      <div fxLayout="column" fxLayoutGap="10px">
-        <span fxLayout="row" fxLayoutGap="10px" *ngIf="org?.email">
-          <mat-icon>email</mat-icon>
-          <span><a [href]="'mailto:' + org.email" target="_top">{{ org.email }}</a></span>
-        </span>
-        <span fxLayout="row" fxLayoutGap="10px" *ngIf="org?.link">
-          <mat-icon>link</mat-icon>
-          <span><a [href]="org.link" target="_blank">{{ org.link }}</a></span>
-        </span>
-        <span fxLayout="row" fxLayoutGap="10px" *ngIf="org?.location">
-          <mat-icon>location_on</mat-icon>
-          <span class="ellipsis-lines">{{ org.location }}</span>
-        </span>
-        <p>Markdown description placeholder</p>
+  <div class="container" *ngIf="(organization$ | async) as org" fxLayout="column" fxLayoutGap="10px">
+    <mat-card fxFlex class="my-3">
+      <div fxLayout="row" fxLayoutGap="10px">
+        <div>
+          <img src="https://via.placeholder.com/150" alt="Logo" class="img-circle">
```

Comment:
Might be a good github issue to split out and get a gravatar for this.

Reply:
https://github.com/ga4gh/dockstore/issues/2052